src/python/turicreate/_deps/__init__.py

```diff
@@ -9,7 +9,17 @@
 from distutils.version import StrictVersion as _StrictVersion
 import logging as _logging
 import re as _re
-from turicreate._deps.minimal_package import is_minimal_pkg
+from turicreate import __version__
+from turicreate._deps.minimal_package import (
+    is_minimal_pkg,
+    is_py_interactive,
+)
+
+if is_py_interactive() and is_minimal_pkg():
+    print(
```

Comment:
When do the two print statements in this file get printed out? When the user imports TuriCreate if they have a minimal install?

Reply:
when they use this minimal package in interactive mode.

Reply:
I don't think that makes sense. I think we should remove these print statements and instead only print out these messages if they try to use some of the non-minimal functionality.

Reply:
Sure. I follow whatever team decides.

Reply:
Toby is right.  The users should only know this is the minimal package if they try to use some of the non-mimimal functionality.  Otherwise this will be a really bad experience for users like trove.

Reply:
This will only print when they use interactive mode during first-time import. So it will only print once. But anyway,